docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -141,6 +143,97 @@ You can copy the preceding code block into a file called `yba-values.yaml` and t
 
 If you are looking for a customization which is not listed, you can view all the supported options and their default values by running the `helm show values yugabytedb/yugaware --version {{<yb-version version="preview" format="short">}}` command and copying the specific section to your own values file.
 
+### Use Yugabyte Kubernetes Operator to automate YBA deployments
+
+The Yugabyte Kubernetes Operator {{<badge/tp>}} simplifies the deployment and operations of YugabyteDB clusters in Kubernetes environments. This operator automates tasks such as provisioning, scaling, and managing lifecycle events. A typical use case involves deploying a scalable YugabyteDB cluster within a private Kubernetes cluster to support high-availability applications.
+
+To install YugabyteDB Anywhere (YBA) and a universe using the Yugabyte Kubernetes Operator, do the following:
```

Comment:
We should add that for operator to work correctly we need the rbac.create=true setting, since the operator needs clusterroles as it creates its own providers.